### DISCUSSION TITLE
specify ALL index.html / index.pug as entry points

### DISCUSSION THREAD

**Discussion by cekvenich2** [parcel-bundler/parcel#6793] [Q&A]
How to specify all as entry points? A bit like snowpack.

**Comment by mischnic** [comment]
Something like `parcel build 'src/**/*.html'`?

**Comment by cekvenich2** [comment]
Yes please. Does that work?

**Comment by cekvenich2** [comment]
With *.pug?

**Reply by mischnic** [reply]
Yes, it should

**Reply by aakoch** [reply]
Yes. I use something close: `parcel build ./src/index.pug './src/**/index.pug'`. I'm not sure if the first one is even necessary.

**Comment by cekvenich2** [comment]
THANKS. Got it working: https://github.com/axthe/axthe/blob/main/static/package.json
No need for first src.
It helped when I switched to v2.x of parcel. 
Still root pug will not refresh, but .js and subfolder pug does.

@aakoch THANK again.
